src/main/java/be/minelabs/entity/BohrBlueprintEntity.java

```diff
@@ -330,12 +335,16 @@ private void launchParticle(Item item) {
         world.spawnEntity(entity);
     }
 
-    private void decomposeAtom() {
-        for (int p = 0; p < getProtons(); p++) launchParticle(Items.PROTON);
-        for (int n = 0; n < getNeutrons(); n++) launchParticle(Items.NEUTRON);
-        for (int e = 0; e < getElectrons(); e++) launchParticle(Items.ELECTRON);
 
-        clear();
+    private void decomposeAtom() {
```

Comment:
An idea: even if the bohr plate is broken, we can have it launch particles (it becomes unstable because the bohr plate is the thing keeping it stable in the world).

Reply:
Outside of PR scope